### ISSUE TITLE
Proton directory correctly recognized and chosen but then  not found as I launch a game

### ISSUE THREAD

**Issue by NotesOfReality** [flavioislima/HeroicGamesLauncher#194]
**Describe the bug**
Heroic Games Launcher (alias: HGL) correctly recognizes the "Proton 5.13" directory in `~/.config/heroic/tools/proton` as shown in the following screenshot ![HGL_1](https://user-images.githubusercontent.com/11523796/108635163-2cb9af80-747e-11eb-9b8d-8052b7fb0cd8.png) , but as I launch a game it fails.
The log reports `FileNotFoundError: [Errno 2] No such file or directory: '/home/nor/.config/heroic/tools/protonProton 5.13/proton'
[36966] Failed to execute script cli`, full log [here](https://dogbin.f0x1d.com/uculirewog).

**To Reproduce**
1. Download whatever proton release you like
2. Copy its directory (the one with the `proton` binary file etc...) to `~/.config/heroic/tools/proton` 
3. Launch HGL and set (even globally if you want) the "Wine Version" to your version of proton
4. Launch any game following the previous setting, HGL will fail at launching it
5. Check the "Latest Log" by the settings(gear icon) menu of the game you launched

**Expected behavior**
Just HGL launching the game correctly, which (obviously?) happens if I rename the proton directory to `protonProton 5.13` and I move it to `~/.config/heroic/tools/proton`.

**Desktop (please complete the following information):**
 - OS: Pop!_OS 20.10
 - Version: 1.5.3
-  Game: Enter The Gungeon (codename: Garlic)
-  Wine/Proton version: Proton 5.13 

**Additional context**
Eh maybe not a context but just a typo(?): the list of wine and proton directories (checked by HGL) which is shown by the "Help" button in the setting is partially incorrect, HGL doesn't check `~/.config/heroic/Tools/wine` nor `~/.config/heroic/Tools/proton`, it checks `~/.config/heroic/tools/wine` and `~/.config/heroic/tools/proton`.
![HGL_2](https://user-images.githubusercontent.com/11523796/108635612-87540b00-7480-11eb-8958-288fd2c7ca03.png)

**Comment by Goggo66** [comment]
Please see #155 for a related report.

**Comment by NotesOfReality** [comment]
My bad, I checked the opened issues but I forgot to check the closed ones.
I'm sorry for bothering people here with my forgetfulness.
So *if* I've understood it correctly I should put a symlink in `~/.config/heroic/Tools/proton` to the actual proton directory?

**Comment by Goggo66** [comment]
I tried that as well, and it didn't work.
What I have done is to create a symlink at .local/share/Steam/steamapps/common (which didn't exist on my system) to the folder containing the Proton release(s) as downloaded e.g. via Steam.

**Comment by NotesOfReality** [comment]
Wait then there's an actual bug with the `~/.config/heroic/Tools/proton` folder implementation?

I'll workaround things as you suggested anyway, thanks.

**Comment by Goggo66** [comment]
Idk, maybe?

Looking at the code in utils.ts:

```
const protonPaths: string[] = [`${heroicToolsPath}/proton`]
const foundPaths = steamPaths.filter((path) => existsSync(path))
```

...

```
  foundPaths.forEach((path) => {
    protonPaths.push(`${path}/steamapps/common/`)
    protonPaths.push(`${path}/compatibilitytools.d/`)
    return
  })`
```

A '/' is appended for the steam paths below, while this is not done for the ${heroicToolsPath}/proton path.
Just a wild guess...

**Comment by flavioislima** [comment]
Hi.

I think it is a bug like @Goggo66 said.
I'll check tonight and fix it on next release. 
I also need to update the help text. Thanks for the catch guys!

**Comment by NotesOfReality** [comment]
Really glad I (accidentally? lol) helped a bit!

**Comment by flavioislima** [comment]
@NotesOfReality true. One of the things I need to figure it out is a beta version to send out to people for help because sometimes I cannot test everything and all distros so then some bug is found only on some specific cases.

If you're interested in testing a version before release, enter on our discord server. the link is on the ReadMe file. :D

**Comment by flavioislima** [comment]
Fixed both things on the main branch now. 
So we will have the fix on the next release.

**Comment by flavioislima** [comment]
Fixed on 1.5.4
